docs/source/system-architecture.md

```diff
@@ -291,6 +299,18 @@ located.
 
 See [Enabling IBM Spectrum Conductor Support](kernel-conductor.html#enabling-ibm-spectrum-conductor-support) for details.
 
+###### CustomResourceProcessProxy
+Enterprise Gateway also provides a implementation of a process proxy derived from KubernetesProcessProxy 
```

Comment:
Can we inclose all of the ProcessProxy class references in back-ticks: `KubernetesProcessProxy`, `CustomResourceProcessProxy`.